usaspending_api/references/views_v2/autocomplete.py

```diff
@@ -106,6 +105,34 @@ def post(self, request):
         return self.agency_autocomplete_response(request)
 
 
+class NAICSAutocompleteViewSet(BaseAutocompleteViewSet):
+
+    def post(self, request):
+        """Return all budget function/subfunction titles matching the provided search text"""
+
+        search_text, limit = self.get_request_payload(request)
+
+        # get relevant TransactionContracts
+        queryset = TransactionContract.objects.filter(naics__isnull=False, naics_description__isnull=False)
```

Comment:
Is it possible for one of the fields to be populated while the other is not? Want to make sure we're not restricting our search more than necessary.